### PR TITLE
Cycle counts port

### PR DESCRIPTION
**Pull request by gerickson808**

### CODE REVIEW

File: admin/views/entity/cyclecountbatchtabs/futurecounts.cfm [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+				<th>Create Physical Count</th>
+			</tr>
+		</thead>
+		<tbody>
+			<cfloop index="local.day" from="1" to="#rc.cyclecountbatch.getRemainingDayCount()#">
+				<cfset local.cycleCountBatchItems = rc.cyclecountbatch.getItemsToCountByDay(local.day,true) />
+				<cfif arrayLen(local.cycleCountBatchItems)>
+					<tr>
+						<td class="primary">#local.day + local.daysCompleted#</td>
+						<cfloop array="#local.cycleCountBatchItems#" item="local.cycleCountBatchItem">
```
Comment (rmarchandTen24) [inline admin/views/entity/cyclecountbatchtabs/futurecounts.cfm]: array and item are not compatible in cf11. please change to index

Comment (gerickson808) [reply admin/views/entity/cyclecountbatchtabs/futurecounts.cfm]: @rmarchandTen24 Updated

File: admin/views/entity/cyclecountbatchtabs/futurecounts.cfm [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+<cfparam name="rc.edit" type="boolean">
+
+<cfset local.cycleCountGroups = rc.cyclecountbatch.getCycleCountGroups() />
+<cfset local.daysCompleted = rc.cyclecountbatch.getPhysicalsCount() />
+<cfoutput>
+	<table class="table table-bordered s-detail-content-table">
+		<thead>
+			<tr>
+				<th>Day</th>
+				<cfloop array="#local.cycleCountGroups#" item="cycleCountGroup">
```
Comment (rmarchandTen24) [inline admin/views/entity/cyclecountbatchtabs/futurecounts.cfm]: @gerickson808  can you do that here as well

File: admin/views/entity/preprocesscyclecountbatch_physicalcount.cfm [hunk 3]
```diff
@@ -54,16 +54,33 @@ Notes:
...
 		<hb:HibachiPropertyRow>
 			<hb:HibachiPropertyList>
 				<hb:HibachiPropertyDisplay object="#rc.processObject#" property="physicalName" edit="#rc.edit#" />
 				<hb:HibachiPropertyDisplay object="#rc.processObject#" property="countPostDateTime" edit="#rc.edit#" />
 			</hb:HibachiPropertyList>
 		</hb:HibachiPropertyRow>
+		<hb:HibachiPropertyRow>
+			<h5>Counts</h5>
+			<hb:HibachiPropertyList>
+			    <cfloop array="#rc.processObject.getCycleCountItems()#" item="local.cycleCountItem" index="index">
```
Comment (rmarchandTen24) [inline admin/views/entity/preprocesscyclecountbatch_physicalcount.cfm]: @gerickson808 and this one

Comment (gerickson808) [reply admin/views/entity/preprocesscyclecountbatch_physicalcount.cfm]: @rmarchandTen24 All set

**Pull request merged**